packages/kit/src/exports/vite/index.js

```diff
@@ -459,6 +460,16 @@ function kit() {
 					});
 				});
 
+				// generate a new manifest that doesn't include prerendered pages
+				fs.writeFileSync(
+					`${paths.output_dir}/server/manifest.js`,
+					`export const manifest = ${generate_manifest({
+						build_data,
+						relative_path: '.',
+						routes: manifest_data.routes.filter((route) => prerender_map.get(route.id) !== true)
```

Comment:
Does `prerender_map` leave out routes which have `export const prerender = 'auto';` (e.g. the "some blog posts static, the long tail dynamic" case)?

Reply:
it's a map of every route ID to `true`, `false` or `'auto'`. this filter will keep `'auto'` routes